### PR TITLE
Jesus/table component

### PR DESCRIPTION
**Pull request by jesus-mdz**
Added Table Component

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -49,16 +53,28 @@ function Header() {
               className={`dropdown-menu ${show ? 'show' : ''}`}
               aria-labelledby="dropdown01"
             >
-              <Link className="dropdown-item" to="/orders">
+              <Link
```
Comment (ahaly92) [inline src/components/Header/Header.js]: these are pretty much the same all the way through. Can we pull out into an array const and loop over to create this dropdown?

Comment (jesus-mdz) [reply src/components/Header/Header.js]: Not a problem!

**Review by mmevans — APPROVED** [review]

File: src/store/product/reducer.js [hunk 2]
```diff
@@ -1,19 +1,37 @@
 import actions from './actionTypes';
 
-const initialState = { productAdded: true };
+let products = JSON.parse(localStorage.getItem('products'));
+const initialState = {
+  productAdded: true,
```
Comment (ahaly92) [inline src/store/product/reducer.js]: you can combine productAdded and productsReceived into one boolean value called loading

File: src/components/Auth/utils/findProduct.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { useSelector } from 'react-redux';
+
+export default (match) => {
```
Comment (ahaly92) [inline src/components/Auth/utils/findProduct.js]: i think the use selector should be pulled out of this util function. This util function should get the array of products and return the one you want.

Comment (ahaly92) [reply src/components/Auth/utils/findProduct.js]: I would also use a ternary operator here if you do that and clean up syntax i don't really see a need for the util function

**Pull request merged**